src/mapper.proto

```diff
@@ -1,24 +1,42 @@
 syntax = "proto3";
 package helium;
 
-message mapper {
-  oneof message {
-    mapper_gps gps = 1;
-    mapper_scan scan = 2;
-    mapper_attach attach = 3;
-  }
+service ScanUpload {
+  rpc Upload(mapper_msg) returns (scan_upload_reply) {}
+}
```

Comment:
it looks like this repo uses `snake_case` by convention. Is there reason this is `PascalCase`?